### PR TITLE
test: add remove-liquidity interface tests

### PR DESCRIPTION
**Pull request by kristiehuang**
## Description
* redirecting 'remove/v2/duplicate/duplicate' to Position Unavailable; '/remove/v2/single' to 404; 'remove/:nonexistent-pool' to Position Unavailable
* adding test cases that they redirect to 404 or error pages accordingly
* add 'loading token pair' check for v3

* waiting on updates to cypress-hardhat to support managing liquidity positions, before full hardhat functionality can be tested https://github.com/Uniswap/cypress-hardhat/pull/25

_Linear ticket:_ [WEB-2581](https://linear.app/uniswap/issue/WEB-2581/no-e2e-tests-for-remove-liquidity-fails-for-duplicate-or-single-token)
_Slack thread:_
_Relevant docs:_

### Before
|    Mobile    |   Desktop    |
| ------------ | ------------ |
| paste_before | paste_before |

### After
|    Mobile    |   Desktop   |
| ------------ | ----------- |
| paste_after  | paste_after |

### Reproducing the error

1. 

### QA (ie manual testing)

- [ ] N/A

### Automated testing

- [ ] Unit test
- [ ] Integration/E2E test

### CODE REVIEW

File: src/pages/RemoveLiquidity/V3.tsx [hunk 1]
```diff
@@ -320,7 +320,8 @@ function Remove({ tokenId }: { tokenId: BigNumber }) {
                   <ThemedText.DeprecatedLabel
                     ml="10px"
                     fontSize="20px"
-                  >{`${feeValue0?.currency?.symbol}/${feeValue1?.currency?.symbol}`}</ThemedText.DeprecatedLabel>
+                    id="remove-liquidity-tokens"
+                  >{`${liquidityValue0?.currency?.symbol}/${liquidityValue1?.currency?.symbol}`}</ThemedText.DeprecatedLabel>
```
Comment (kristiehuang) [inline src/pages/RemoveLiquidity/V3.tsx:324]: seems like feeValue hook is flaky in cypress? but feeValue0.currency should be the same as liquidityValue0.currency

File: cypress/e2e/remove-liquidity.test.ts [hunk 2]
```diff
@@ -1,7 +1,35 @@
+import { ChainId, MaxUint256 } from '@uniswap/sdk-core'
+
+import { UNI } from '../../src/constants/tokens'
+
+const UNI_MAINNET = UNI[ChainId.MAINNET]
```
Comment (cbachmeier) [inline cypress/e2e/remove-liquidity.test.ts:5]: nit: you're pulling in the token here but you can just pull in the address directly by using `UNI_ADDRESSES[ChainId.MAINNET]` and importing `UNI_ADDRESSES from '@uniswap/sdk-core'`

File: src/pages/RemoveLiquidity/V3.tsx (hunk 1 above)
Comment (cbachmeier) [reply src/pages/RemoveLiquidity/V3.tsx:324]: what issue were we seeing? I'm hesitant to change working functionality to pass a test

Comment (kristiehuang) [reply src/pages/RemoveLiquidity/V3.tsx:324]: traced it back to this open TODO in useV3PositionFees: https://github.com/Uniswap/interface/blob/5c53d8237d3dfd1e23d192ac499df902503d60a4/src/hooks/useV3PositionFees.ts#L33

may not explicitly be a cypress issue but am seeing this call fail -> catch -> return undefined for `feeValue`

Comment (cbachmeier) [reply src/pages/RemoveLiquidity/V3.tsx:324]: Interesting, okay. I'm fine with this change but can we also change the `DoubleCurrencyLogo` to use `liquidityValue` for consistency?

```
               <DoubleCurrencyLogo
                    currency0={liquidityValue0?.currency}
                    currency1={liquidityValue1?.currency}
                    size={20}
                    margin={true}
                  />

Comment (kristiehuang) [reply src/pages/RemoveLiquidity/V3.tsx:324]: sounds good good catch!

**Review by cartcrom — APPROVED** [review]
Looks great!

**Review by cbachmeier — APPROVED** [review]

**Pull request merged**